frontend/src/app/shared/components/editor/components/ckeditor/op-ckeditor.component.ts

```diff
@@ -209,6 +208,12 @@ export class OpCkeditorComponent implements OnInit, OnDestroy {
         watchdog.editor.on('op:source-code-enabled', () => this.enableManualMode());
         watchdog.editor.on('op:source-code-disabled', () => this.disableManualMode());
 
+        const model = watchdog.editor.model as unknown&{ on:(ev:string, callback:() => unknown) => void };
+        model.on('op:attachment-added', () => document.body.dispatchEvent(new DragEvent('dragend')));
+        model.on('op:attachment-removed', () => document.body.dispatchEvent(new DragEvent('dragend')));
+
+        this.$element.on('dragleave', () => document.body.dispatchEvent(new DragEvent('dragleave')));
```

Comment:
Does this work?

Reply:
Not in 100% of the cases. 

It basically turns the problem a bit around. We receive more `dragleave` then `dragenter` now, when the editing mode is enabled. Hence, when dragging a file in circles over the ckeditor field, the dropboxes vanish. Good thing though is, this state can get repaired, by dragging the file once out of the browser and back into it. It does not stay in the "broken" state, as before.

Reply:
I think I'm too much out of the loop to comment on this, we could talk about it in the design daily if that works for you guys.

Reply:
I added some comments, but there is one thing I cannot manage to solve:

When dragging over the description field, the ckeditor opens up. you can add a delay to it, but I cannot cancel it again. Not sure, what is the reason of this. I get a `dragenter` event on every child of the `displayContainer` of the description field. Hence I also get a lot of activation request (which does not matter, as it will get activated eventually). But I simply cannot cancel them all in an easy and understandable way.